include/openenclave/internal/thread.h

```diff
@@ -54,17 +54,34 @@ bool oe_thread_equal(oe_thread_t thread1, oe_thread_t thread2);
 
 typedef uint32_t oe_once_t;
 
+/* Match the definitions of corelibc/pthread.h */
+#define OE_MUTEX_NORMAL 0
+#define OE_MUTEX_DEFAULT 0
```

Comment:
Since this PR set the default type to recursive, it is easier to understand if both OE_MUTEX_DEFAULT and OE_MUTEX_RECURSIVE have the same value.

> #define OE_MUTEX_DEFAULT OE_MUTEX_RECURSIVE

would read better.

Reply:
Done